packages/test/test-service-load/src/faultInjectionDriver.ts

```diff
@@ -274,6 +274,7 @@ export class FaultInjectionDocumentDeltaConnection
 
 	public goOffline() {
 		this.online = false;
+		this.injectDisconnect();
```

Comment:
This isn't necessarily more accurate since we don't get a signal right away when we lose connection. I think it can take up to 45 seconds for the socket to close due to ping timeout

Reply:
ping won't timeout here, as the real socket is ok, so this will currently never disconnect.